Source/WebCore/Modules/webcodecs/WebCodecsVideoEncoder.cpp

```diff
@@ -156,7 +156,7 @@ ExceptionOr<void> WebCodecsVideoEncoder::configure(ScriptExecutionContext& conte
                 return;
 
             m_isMessageQueueBlocked = true;
-            m_internalEncoder->flush([weakThis = ThreadSafeWeakPtr { *this }, config = WTFMove(config)]() mutable {
+            m_internalEncoder->flush([weakThis = ThreadSafeWeakPtr { *this }, config = WTFMove(config).isolatedCopy(), pendingActivity = takePendingWebCodecActivity()]() mutable {
```

Comment:
why the move here?

Reply:
It allows to use isolatedCopy() && which might save us a string copy if the string ref count is 1.